pkg/blueprint/validate/validate.go

```diff
@@ -21,9 +21,10 @@ import (
 
 	kanister "github.com/kanisterio/kanister/pkg"
 	crv1alpha1 "github.com/kanisterio/kanister/pkg/apis/cr/v1alpha1"
-	_ "github.com/kanisterio/kanister/pkg/function"
 	"github.com/kanisterio/kanister/pkg/param"
 	"github.com/kanisterio/kanister/pkg/utils"
+
+	_ "github.com/kanisterio/kanister/pkg/function"
```

Comment:
does this need to be grouped separately, it's also a kanister dep right?

Reply:
yes, it is grouped as blank in gci linter.